### PR TITLE
Fix made to MKE CLI install using script code-block.

### PR DESCRIPTION
**Pull request by KoryKessel-Mirantis**
Edit made in reference to issues #67, #72, and #73, to fix link in code-block of Install MKE CLI with script.

### CODE REVIEW

**Review by prashshr — APPROVED** [review]
LGTM

**Review by vivek — CHANGES_REQUESTED** [review]
Its changed only at one place, there are two more links there, please change them there as well.

**Comment by ry4nz** [comment]
LGTM now

**Review by vivek — CHANGES_REQUESTED** [review]
Changes LGTM. 

Can we also update references `v4.0.0-alpha.2.0` to `v4.0.0-alpha.4.0`?

Couple other suggestions:

* Update links under `Install manually` with them pointing to change `v4.0.0-alpha.2.0` to `v4.0.0-alpha.4.0`
* Move `Install using script` before `Install manual` as installation using script is preferred way to install `mkectl`

**Review by vivek — APPROVED** [review]

File: content/docs/getting-started/install-MKE-CLI.md [hunk 1]
```diff
@@ -122,4 +100,27 @@ underlying architecture, based on which it will install the `k0sctl`, `kubectl`
...
+| Linux        | arm64        | [download](https://s3.us-east-2.amazonaws.com/packages-stage-mirantis.com/v4.0.0-alpha.4.0/mkectl_linux_arm64.tar.gz) |
+| Linux        | x86_64       | [download](https://s3.us-east-2.amazonaws.com/packages-stage-mirantis.com/v4.0.0-alpha.4.0/mkectl_linux_x86_64.tar.gz) |
+| MacOS        | arm64        | [download](https://s3.us-east-2.amazonaws.com/packages-stage-mirantis.com/v4.0.0-alpha.4.0/mkectl_darwin_arm64.tar.gz) |
+| MacOS        | x86_64       | [download](https://s3.us-east-2.amazonaws.com/packages-stage-mirantis.com/v4.0.0-alpha.4.0/mkectl_darwin_x86_64.tar.gz) |
+| Windows      | arm64        | [download](https://s3.us-east-2.amazonaws.com/packages-stage-mirantis.com/v4.0.0-alpha.4.0/mkectl_windows_arm64.zip) |
+| Windows      | x86_64       | [download](https://s3.us-east-2.amazonaws.com/packages-stage-mirantis.com/v4.0.0-alpha.4.0/mkectl_windows_x86_64.zip) |
+
+Envisioned as a single binary that is capable of managing MKE clusters without
+any additional dependencies, the MKE CLI also requires that you have the
+following tools installed on your system:
```
Comment (MagdaDziadosz) [inline content/docs/getting-started/install-MKE-CLI.md:120]: Could you rephrase the fragment to read more technical, currently it is a bit too commercial Maybe:
MKE CLI is a single binary that is capable of managing MKE clusters without
any additional dependencies. To use it, you need to install the following tools on your system:

File: content/docs/getting-started/install-MKE-CLI.md [hunk 2]
```diff
@@ -3,56 +3,34 @@ title: Install the MKE CLI
...
+automatically, using the script provided in the section that immediately
+follows, or you can do it manually.
 
 ## Install using script
 
-To automatically install the necessary dependencies, you can utilize the `install.sh` script as shown in the example below.
-
-**Procedure:**
+To automatically install the necessary dependencies, you can use the
+`install.sh` script offered in the example procedure that follows.
```
Comment (MagdaDziadosz) [inline content/docs/getting-started/install-MKE-CLI.md:14]:
```suggestion
Install the necessary dependencies automatically, you can use the
`install.sh` script shown in the example:
```

File: content/docs/getting-started/install-MKE-CLI.md [hunk 3]
```diff
@@ -3,56 +3,34 @@ title: Install the MKE CLI
...
-tools installed on your system as well:
-
-| Tool     | Version          | Download |
-|----------|------------------|----------|
-| kubectl  | 1.29.0 or later  | [download](https://kubernetes.io/docs/tasks/tools/#kubectl) |
-| k0sctl   | 0.17.0 or later but less than `0.18.0` | [download](https://github.com/k0sproject/k0sctl/releases) |
+Before you can proceed with the MKE installation, you must download and install
+`mkectl`, the MKE CLI tool, as well as `kubectl` and `k0sctl`. You can do this
+automatically, using the script provided in the section that immediately
+follows, or you can do it manually.
```
Comment (MagdaDziadosz) [inline content/docs/getting-started/install-MKE-CLI.md:9]:
```suggestion
`mkectl`, the MKE CLI tool, as well as `kubectl` and `k0sctl`. You can do this
[automatically](#install-using-script), using the provided script, or you can do it [manually](#install-manually) .
```

File: content/docs/getting-started/install-MKE-CLI.md [hunk 4]
```diff
@@ -3,56 +3,34 @@ title: Install the MKE CLI
...
 
 ## Install using script
```
Comment (MagdaDziadosz) [inline content/docs/getting-started/install-MKE-CLI.md:11]: Should'n it be titled “Install automatically” if the next one is “Install manually”?

File: content/docs/getting-started/install-MKE-CLI.md (hunk 1 above)
Comment (KoryKessel-Mirantis) [reply content/docs/getting-started/install-MKE-CLI.md:120]: Good thought. I tweaked the text to read:

"The MKE CLI is a single binary that is capable of managing MKE clusters without any additional dependencies. Its use, though, requires the presence of the following tools on your system:"

File: content/docs/getting-started/install-MKE-CLI.md (hunk 2 above)
Comment (KoryKessel-Mirantis) [reply content/docs/getting-started/install-MKE-CLI.md:14]: Tweaked to read as follows:

"To automatically install the necessary dependencies, you can use an `install.sh` script, as exemplified in the following procedure:"

The reason being that the procedure is not actually an _example_.
